plugins/modules/ec2_asg.py

```diff
@@ -87,6 +87,12 @@
       - 'See also U(https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-autoscaling-autoscalinggroup-mixedinstancespolicy.html)'
     required: false
     suboptions:
+      instance_requirements:
+        description:
+        - A set of params describing the specs instances should have
+        type: dict
+        required: false
+        version_added: 3.2.0
```

Comment:
```suggestion
        version_added: 3.3.0
```

Reply:
Since this introduces a breaking change, I think version_added should be 4.0.0. @markuman

Reply:
As a non-required suboption does this break existing playbooks?  If so, IMO it would be better to re-write this PR than simply break things with 4.0.0

Reply:
Oh yes. breaking change must be set to 4.0.0.

Reply:
@tremble  The breaking change is related to the return value.  

Another possibility is to split it.  
* ship just the suboption with 3.3.0
* ship the change on the return value in 4.0.0

Reply:
I'm not sure it's worth having the 'breaking' part of this change at all.  There's no clean way to handle a return value completely changing in type.  It's one thing to drop something where an alternative's been offered for multiple-releases, it's another to have **no** clean upgrade path.

Reply:
https://github.com/ansible-collections/community.aws/pull/232#discussion_r490084475